TicTacToe/CLI.cs

```diff
@@ -2,11 +2,31 @@
 
 namespace TicTacToe
 {
-    public class CLI
+    public class CLI : IUserInput
     {
         public void LogToConsole(string message)
         {
             Console.WriteLine(message);
         }
+
+        public void WelcomeToGame()
+        {
+            this.LogToConsole("Welcome to Tic Tac Toe!\n\nTo start a new game, please enter both players' names\n\n");
+        }
+
+        public string[] GetPlayerNames()
+        {
+            Console.WriteLine("Player 1 (X), please enter your name:");
+            string player1Name  = Console.ReadLine();
+            Console.WriteLine("Player 2 (O), please enter your name:");
```

Comment:
I'm noticing these "X" and "O"'s throughout your codebase (tests and core code). What's going to happen if we change ask to change these markers? How can you adjust the code so you would only need to update it in as few places as possible?

Reply:
Thanks, I'll refactor to define the marker characters somewhere (either in Game or Player), and then use the variable (or object property throughout)